diploma-project/components/ItemsScrollWithArrows.jsx

```diff
@@ -0,0 +1,96 @@
+import {useCallback, useEffect, useRef, useState} from "react";
+import {ChevronLeft, ChevronRight} from "react-feather";
+
+const ItemsScrollWithArrows = ({categoryName = "", listOfItems, message = ""}) => {
+    const [curr, setCurr] = useState(0);
+    const [translation, setTranslation] = useState(null);
+    const [timesToScroll, setTimesToScroll] = useState(null);
+    const containerRef = useRef(null);
+    const spacingInPixels = 20;
+
+    const prev = useCallback(() => setCurr((curr) => (curr === 0 ? 0 : curr - 1)), []);
+    const next = useCallback(() => setCurr((curr) => (curr === listOfItems.length - 1 ? listOfItems.length - 1  : curr + 1)), [listOfItems.length]);
+
+    const resetToFirst = () => setCurr(0);
+
+    useEffect(() => {
+        const updateCardWidth = () => {
+            if(listOfItems.length === 0) {
+                setTranslation(null);
+            } else if (containerRef.current) {
+                const cardWidth = containerRef.current.querySelector("#item-0").getBoundingClientRect().width;
```

Comment:
In React, it's generally better to avoid using IDs and instead use refs for selecting elements.